src/app/client/src/app/modules/shared-feature/components/tnc-popup/terms-conditions-popup.component.ts

```diff
@@ -75,7 +75,7 @@ export class TermsAndConditionsPopupComponent implements OnInit, OnDestroy {
       }
     };
     if (this.userService.userProfile.managedBy) {
```

Comment:
this check is not required

Reply:
we have confirmed with backed team we should only send the user id for child users only.
If we need to send user id always(for child and managed users) we might need to change SSO sign-up and self sign up flows as well to send user id always to have consistency.